src/lib/components/no-result/terra-no-result-notice.component.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { TerraNoResultNoticeComponent } from './terra-no-result-notice.component';
+import { TerraButtonComponent } from '../..';
+import { By } from '@angular/platform-browser';
+import { noResultsNoticeMockButtons } from '../../testing/mock-buttons';
+import { Component, DebugElement } from '@angular/core';
+
+@Component({
+    selector: 'terra-button',
+    template: ''
+})
+class TerraButtonMockComponent extends TerraButtonComponent {}
+
+describe('TerraNoResultNoticeComponent', () => {
+    let fixture: ComponentFixture<TerraNoResultNoticeComponent>;
+    let component: TerraNoResultNoticeComponent;
+
+    beforeEach(() => {
+        fixture = TestBed.configureTestingModule({
+            declarations: [TerraButtonMockComponent, TerraNoResultNoticeComponent]
+        }).createComponent(TerraNoResultNoticeComponent);
+        component = fixture.componentInstance;
+
+        fixture.detectChanges();
+    });
+
+    it('should create an instance', () => {
+        expect(component).toBeTruthy();
+    });
+
+    it('should highlight inner buttons', () => {
+        component.inputButtons = noResultsNoticeMockButtons;
+        fixture.detectChanges();
+        const buttons: Array<DebugElement> = fixture.debugElement.queryAll(By.css('terra-button'));
+        let buttonComponent: TerraButtonComponent = buttons[0].componentInstance as TerraButtonComponent;
+        fixture.detectChanges();
+        expect(buttonComponent.inputIsHighlighted).toBe(true);
```

Comment:
Du willst ja testen ob *alle*  buttons diese Bedingung erfüllen, oder?
Dann geht das so:
```suggestion
        expect(buttons.every((button:DebugElement) => (button.componentInstance as TerraButtonComponent).inputIsHighlighted)).toBe(true);
```